monthly_scheduler/__init__.py

```diff
@@ -0,0 +1,119 @@
+import logging
+import os
+import requests 
+from datetime import datetime, timezone 
+from typing import List, Dict, Optional
+from tenacity import retry, stop_after_attempt, wait_exponential
+import azure.functions as func
+# logger setting 
+logging.basicConfig(level=logging.INFO) 
+logger = logging.getLogger(__name__)
+
+
+""" 
+Monthly Scheduler: 
+1. The trigger will run at 8am EST every 1st date of the month 
+2. It will loop through all the reports in the MONTHLY_REPORTS list 
+3. Then it will create a report for each monthly report type using the /api/reports/generate/curation endpoint 
+4. Once done, the report will be uploaded to blob storage 
+
+To do:
+5. trigger the email function to an email to send to admin the link to the report that has been created 
+6. automatically send the report to user using the blob link got from the curation endpoint 
+check other way to use endpoint 
+
+"""
+
+MONTHLY_REPORTS = ['Monthly_Economics'] # todo: add ecommerce report
+
+CURATION_REPORT_ENDPOINT = f'{os.environ["WEB_APP_URL"]}/api/reports/generate/curation'
+
+EMAIL_ENDPOINT = f'{os.environ["WEB_APP_URL"]}/api/reports/digest'
+
+TIMEOUT_SECONDS = 300
+
+RECIPIENTS = ['namtran6701@gmail.com']
```

Comment:
In order to test this accurately, we need to be able to change this email or add more emails, can you add a cosmos query to get emails from database? (create a new table for this, named recipients with name, email, and active status)